commands/wiki.js

```diff
@@ -1,5 +1,128 @@
+	},
+	"participate": {
+		name: "Participate",
+		url: "https://github.com/official-stockfish/Stockfish/wiki/Developers#participating-in-the-project",
+		description: `Stockfish's improvement over the last decade has been a great community effort. There are a few ways to help contribute to its growth.`,
+	},
+	"download": {
+		name: "Download",
+		url: "https://github.com/official-stockfish/Stockfish/wiki/Download-and-usage",
+		description: `Stockfish binary downloads for different systems and architectures.`,
+	},
+	"gui": {
+		name: "GUI",
+		url: "https://github.com/official-stockfish/Stockfish/wiki/Download-and-usage#download-a-chess-gui",
+		description: `Good graphical user interfaces to use the Stockfish chess engine with.`,
+	},
+	"governance": {
+		name: "Governance",
+		url: "https://github.com/official-stockfish/Stockfish/wiki/Governance-and-responsibilities",
+		description: `Governance and responsibilities of various individuals in different Stockfish projects.`,
+	},
+	"regression_tests": {
+		name: "Regression Tests",
+		url: "https://github.com/official-stockfish/Stockfish/wiki/Regression-Tests",
+		description: ``,
+	},
+	"current_dev": {
+		name: "Current Development",
+		url: "https://github.com/official-stockfish/Stockfish/wiki/Regression-Tests#current-development",
+		description: ``,
+	},
+	"faq": {
+		name: "FAQ",
+		url: "https://github.com/official-stockfish/Stockfish/wiki/Stockfish-FAQ",
+		description: `Frequently asked questions pertaining to various Stockfish projects.`,
+	},
+	"terminology": {
+		name: "Terminology",
+		url: "https://github.com/official-stockfish/Stockfish/wiki/Terminology",
+		description: `List of frequently used terminolory related to engine development.`,
+	},
+	"depth_vs_tc": {
+		name: "Depth vs TC",
+		url: "https://github.com/official-stockfish/Stockfish/wiki/Useful-data#depth-vs-tc",
+		description: ``,
+	},
+	"elo_hash": {
+		name: "Elo Hash",
+		url: "https://github.com/official-stockfish/Stockfish/wiki/Useful-data#elo-cost-of-small-hash",
+		description: ``,
+	},
+	"elo_syzygy": {
+		name: "Elo syzygy",
+		url: "https://github.com/official-stockfish/Stockfish/wiki/Useful-data#elo-gain-using-syzygy",
+		description: ``,
+	},
+	"threading_effeciency": {
+		name: "Threading Efficiency",
+		url: "https://github.com/official-stockfish/Stockfish/wiki/Useful-data#threading-efficiency-and-elo-gain",
+		description: ``,
+	},
+	"elo_speedups": {
+		name: "Elo Speedups",
+		url: "https://github.com/official-stockfish/Stockfish/wiki/Useful-data#elo-from-speedups",
+		description: ``,
+	},
+	"time_odds": {
+		name: "Time Odds",
+		url: "https://github.com/official-stockfish/Stockfish/wiki/Useful-data#elo-gain-with-time-odds",
+		description: ``,
+	},
+	"fishtest": {
+		name: "Fishtest",
+		url: "https://github.com/glinscott/fishtest",
+		description: ``,
+	},
+	"nnue_pytorch": {
+		name: "NNUE Pytorch",
+		url: "https://github.com/glinscott/nnue-pytorch",
+		description: ``,
+	}
```

Comment:
dunno where to add this, did you try this with the broken links ?

Reply:
Which broken links?

Reply:
the original had some commented choices, node.js didnt want to run with them for whatever reason.